package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "license": "MIT",
   "engines": {
-    "node": "^12.16.3"
+    "node": "^12.18.4"
```

Comment:
This can only be 12.18.3 because that’s the most recent version the PaaS supports. From https://groups.google.com/a/digital.cabinet-office.gov.uk/g/gov-uk-paas-announce/c/aiF05_XAH8I:

```
## nodejs_buildpack: v1.7.26

* node:
    * Removed: 10.20.1, 12.18.0, 14.4.0, 14.5.0
    * Added: 12.18.3, 14.7.0, 14.8.0, 10.22.0
```

Reply:
Done